src/components/frame/v5/pages/UserProfilePage/partials/Row/Row.tsx

```diff
@@ -7,6 +7,7 @@ import { type RowProps } from './types.ts';
 
 const displayName = 'v5.pages.UserProfilePage.partials.Row';
 
+// @TODO remove this and refactor UserAccountForm like UserPreferencesPage was
```

Comment:
This would be great. Spreading the items here is throwing errors in the console.

Reply:
➕ 1